Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/entrycard.vue

```diff
@@ -1,11 +1,16 @@
 <script lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faComment } from "@fortawesome/free-regular-svg-icons";
```

Comment:
We don't use regular icons a lot, normally we use solid icons. Here is 1 of the 2 scenarios that we use regular icon.